etcdserver/api/v3election/election.go

```diff
@@ -16,12 +16,17 @@ package v3election
 
 import (
 	"context"
+	"errors"
 
 	"github.com/coreos/etcd/clientv3"
 	"github.com/coreos/etcd/clientv3/concurrency"
 	epb "github.com/coreos/etcd/etcdserver/api/v3election/v3electionpb"
 )
 
+// ErrMissingLeaderKey is returned when election API is
+// missing "leader" field.
+var ErrMissingLeaderKey = errors.New(`missing "leader" field`)
```

Comment:
the error message also needs to be fixed.

Reply:
Oh I meant,

```go
type ProclaimRequest struct {
	// leader is the leadership hold on the election.
	Leader *LeaderKey `protobuf:"bytes,1,opt,name=leader" json:"leader,omitempty"`
	// value is an update meant to overwrite the leader's current value.
	Value []byte `protobuf:"bytes,2,opt,name=value,proto3" json:"value,omitempty"`
}
```

So, the JSON field is still `"leader"`

Reply:
oh. ok.

Reply:
the error message can be more descriptive like `leader field must be provided`